airflow/www/views.py

```diff
@@ -719,6 +718,66 @@ def rendered(self):
             form=form,
             title=title, )
 
+    @expose('/get_logs_with_metadata')
+    @login_required
+    @wwwutils.action_logging
+    @provide_session
+    def get_logs_with_metadata(self, session=None):
```

Comment:
Not putting the endpoint into API to reuse the dagbag. Open for discussion on this.